### PR TITLE
Accessibility review

### PR DESCRIPTION
**Pull request by chef-louis**
#40.

@pm-moyanor Feel free to suggest your own accessibility changes if you see any.

### CODE REVIEW

**Review by qisforq — COMMENTED** [review]
I see @pm-moyanor is the only requested reviewer for this PR so I won't approve, but I did look over and it looks good to me

**Comment by qisforq** [comment]
> I see @pm-moyanor is the only requested reviewer for this PR so I won't approve, but I did look over and it looks good to me

Actually now I am popping up as a reviewer 🤔. @pm-moyanor if all looks good to you I'll go ahead and merge

**Comment by chef-louis** [comment]
Yah I made @pm-moyanor an explicit reviewer because we were working on this ticket together.

@qisforq you may have become a reviewer after posting a comment? Either way, I'm all for pushing this through but did want to give Paula a chance to review

**Comment by pm-moyanor** [comment]
looks good @chef-louis 👍🏻

File: src/components/common/Header.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const Header = () => {
   };
 
   return (
-    <>
+    <header>
```
Comment (qisforq) [inline src/components/common/Header.tsx:62]: @chef-louis I just noticed that some of the changes in this PR aren't playing nice with eslint.

Eslint wants us to wrap all JSX expressions in a JSX fragment (<></>).

I'll add this in my next PR, I just wanted to alert you. If you're not seeing that warning when viewing Header.tsx, we should make sure the eslint plugin is working correctly for you

Comment (qisforq) [reply src/components/common/Header.tsx:62]: Actually ignore this - the warning doesn't pop up for me when after pulling all new changes - not sure what was going on before

**Pull request merged**